deploys/demo.syncfor.science/config/defaults/vendors/epic.yml

```diff
@@ -0,0 +1,11 @@
+---
+api:
+  url: https://open-ic.epic.com/Argonaut/api/FHIR/Argonaut/
+  patient: Tbt3KuCY0B5PSrJvCu2j-PlK.aiHsu2xUjUM8bWpetXoB
+
+auth:
+  strategy: refresh_token
+  client_id: ad3fe986-cbad-4fcc-a487-3e9ee1ef3a4d
+  client_secret: argonaut
+  scope: launch/patient patient/*.read offline_access
+  basic: yes
```

Comment:
Random comment: Rather than `basic: yes` would probable be clearer to say `confidential_client: true` across the board.